fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -165,6 +165,9 @@ def __init__(
                 self.buffer_max_size / 2 ** 20, model_size / 2 ** 20
             )
         )
+        if dist.get_world_size(self.process_group) == 1:
```

Comment:
Should the above calculation and logging be inside an if clause?

Reply:
I can flip the ordering, probably a bit cleaner log wise you're right (easier to understand at least)